master/internal/task/allocation.go

```diff
@@ -626,9 +626,8 @@ func (a *allocation) resourcesAllocated(msg *sproto.ResourcesAllocated) error {
 	err = db.RecordTaskStats(context.TODO(), &model.TaskStats{
 		AllocationID: msg.ID,
 		EventType:    "QUEUED",
-		// allocation.startTime?
-		StartTime: &msg.JobSubmissionTime,
-		EndTime:   &now,
+		StartTime:    &msg.JobSubmissionTime,
```

Comment:
hmm. i think JobSubmissionTime is wrong. if anything the language here is wrong. "task stats" implies this is talking about the task level to me, but job submission time is when the experiment was submitted.

Reply:
yeah, I think there's more to look at here but it isn't immediately obvious to me that'd be a different level of wrong than the current state of "we put time.zero as the start of queued period for k8s". Does it make sense to work on that on a separate ticket w/ a different priority?

Reply:
yeah, they're equally totally wrong, so i'd lean to just fixing it. i wouldnt call this commit "fix: k8s rp queued stats" if you do it in two PRs.

Reply:
https://github.com/determined-ai/determined/pull/9028#issuecomment-2032991065